app/components/shared/FormMarkdownEditorField.js

```diff
@@ -141,7 +140,6 @@ class FormMarkdownEdtiorField extends React.Component {
     e.preventDefault();
 
     this.markdownEditor.bold();
-    autoresizeTextarea(this.textarea);
```

Comment:
You gotta trigger a manual resize here if you can, since we've altered the text inside the textarea, there's a chance that it's bumped the content down onto another line (which would need to change the height of the textarea)